packages/server/src/schema/operations.ts

```diff
@@ -0,0 +1,14 @@
+import { Resolver } from './resolvers';
+import * as resolvers from './resolvers';
+
+export const createSimulation = createResolve(resolvers.createSimulation);
+
+export const given = createResolve(resolvers.given);
+
+function createResolve(resolver: Resolver<unknown, unknown>) {
+  return {
+    resolve(_: any, args: any, context: any): Promise<any> {
+      return resolver.resolve(args, context);
+    }
+  };
+}
```

Comment:
We do this here so that we can have strongly typed resolvers and subscribers, but when they get plugged into the schema they're loosy goosy enough that we don't have to fiddle with nexus typing.